.github/workflows/pr-title-linter.yml

```
@@ -0,0 +1,16 @@
+name: "Conventional Commit Validation PR Title"
```

nit:

Most of out github actions sentences, like

Remove labels after issue (or PR) closed
Close stale issues and PRs
Trigger MetaMask Desktop CI
Update LavaMoat policies

So maybe we do that here, too?


```suggestion
name: "Validation PR Title adheres to Conventional Commits"
```

Also, the action's yml file name generally follow the `name` property. So it wouldn't hurt to rename it to resemble whatever action `"name"` you end up with.